_includes/releases/v23.1/v23.1.0-rc.1.md

```diff
@@ -0,0 +1,78 @@
+## v23.1.0-rc.1
+
+Release Date: May 2, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-0-rc-1-sql-language-changes">SQL language changes</h3>
+
+- The [`crdb_internal.transaction_contention_events` job](../v23.1/crdb-internal.html) now succeeds even if an error occurs getting the name of a non-SQL key. [#101871][#101871]
```

Comment:
This is not a job afaik, it is a system table. 

Also, I think this belongs under Bug fixes.

```suggestion
- The [`crdb_internal.transaction_contention_events` table](../v23.1/crdb-internal.html) now generates even if an error occurs getting the name of a non-SQL key. [#101871][#101871]
```

Reply:
You're right, but I've put it under DB Console changes. When I was editing this, I included some info from a different item -- this is just a performance improvement.